packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -44,6 +45,17 @@ export class EpochTracker {
         return this._fluidEpoch;
     }
 
+    public async validateEpochFromPush(details: IConnected) {
+        const epoch = (details as any).epoch;
```

Comment:
I assume this is short-cut for adding epoch to IConnected, Can we define and use IOdspConnected with extra field everywhere in ODSP code? I think as long as it derives from IConnected, tsc should be happy

Reply:
I am adding it in IConnected interface. https://github.com/microsoft/FluidFramework/pull/4935

Reply:
We should run it by @tanviraumi . I think there is some benefit from not adding everything into common interface for all drivers, and keeping it in ODSP. That said, epoch (in some form or shape) is likely to eventually show up for all end-points, as data center restore is something that can happen anywhere :)